src/v2/api/controllers/UserController.ts

```diff
@@ -56,6 +56,7 @@ import { UpdateSuperheroDTO } from '../dtos/UpdateSuperheroDTO';
 import { DisciplineIdsResponse } from '../responses/DisciplineResponse';
 import { SuperheroResponse } from '../responses/SuperheroResponse';
 import { ApiEndpoint } from 'src/v2/utils/documentation/decorators';
+
 @ApiTags('User')
```

Comment:
Line 382. It would be better to write 'Id of the role of the user'

Reply:
Not my endpoint